client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -1,185 +1,129 @@
-									{
-										components: {
-											strong: <strong />,
-										},
-									}
-								) }
-								<div className="site-migration-instructions__migration-key">
-									<code className="site-migration-instructions__key">
-										{ hideSiteMigrationKey
-											? '*'.repeat( siteMigrationKey.length - 5 )
-											: siteMigrationKey }
-									</code>
-									<Button
-										onClick={ () => {
-											setHideSiteMigrationKey( ! hideSiteMigrationKey );
-										} }
-										className="site-migration-instructions__show-key-button components-button"
-									>
-										{ hideSiteMigrationKey ? translate( 'Show key' ) : translate( 'Hide key' ) }
-									</Button>
-									<ClipboardButton
-										text={ siteMigrationKey }
-										className="site-migration-instructions__copy-key-button is-secondary"
-										onCopy={ onCopy }
-									>
-										{ buttonTextCopy ? translate( 'Copied!' ) : translate( 'Copy key' ) }
-									</ClipboardButton>
-								</div>
-							</>
-						) }
-					</div>
-				</li>
 				<li>
 					{ translate(
-						'Go to the {{a}}Migrate Guru page on the source site{{/a}}, enter your email address, and click {{strong}}Migrate{{/strong}}.',
+						'Install and active the {{a}}Migrate Guru plugin{{/a}} on your existing site.',
 						{
 							components: {
-								a: <a href={ sourceSiteUrl } target="_blank" rel="noreferrer" />,
-								strong: <strong />,
+								a: <a href={ getInstallationURL( fromUrl ) } target="_blank" rel="noreferrer" />,
 							},
 						}
 					) }
 				</li>
 				<li>
 					{ translate(
-						'You will see a screen showing multiple hosting providers. Select {{em}}Automattic{{/em}} as the destination host.',
+						'Go to the {{a}}Migrate Guru page on the source site{{/a}}, enter your email address, and click {{strong}}Migrate{{/strong}}.',
 						{
 							components: {
-								em: <em />,
+								a: <a href={ getMigrateGuruPageURL( fromUrl ) } target="_blank" rel="noreferrer" />,
+								strong: <strong />,
 							},
 						}
 					) }
 				</li>
 				<li>
-					{ translate(
-						'Find the {{em}}Migrate Guru Migration Key{{/em}} field, and paste the key you copied in step 2. Then click {{strong}}Migrate{{/strong}} to start your migration.',
-						{
-							components: {
-								em: <em />,
-								strong: <strong />,
-							},
-						}
-					) }
+					{ translate( 'When asked to select a destination host, pick {{em}}“Automattic”{{/em}}.', {
+						components: {
+							em: <em />,
+						},
+					} ) }
 				</li>
-				<li>{ translate( 'Migrate Guru will send you an email when the migration finishes.' ) }</li>
+				{ isSuccess && migrationKey && (
+					<li>
+						{ translate(
+							'Copy and paste the migration key below in the {{em}}“Migrate Guru Migration Key”{{/em}} field and click {{strong}}Migrate{{/strong}}.',
```

Comment:
Question: are the label and button CTA translated in the Blog Vault/Migrate Guru UI?
We run the risk of translating these terms when they are not translated in the underlying screen.

Reply:
Good question!
I checked and saw they were not translating the CTAs, so I applied this change https://github.com/Automattic/wp-calypso/pull/88813/commits/d22bc4dfa381493da09fd3ac2b3f2df5b83a0202 to protect then to be translated.